src/Components/Wizard/Wizard.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import { Wizard as Pf4Wizard } from '@patternfly/react-core';
+import React from 'react';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import { useMutation } from 'react-query';
+
+import { useDispatch } from 'react-redux';
+import { createAWSReservation } from '../../API';
+import { useWizardContext } from '../Common/WizardContext';
+
+const Wizard = ({ steps, onNext, isOpen, onClose, imageID, ...props }) => {
+  const [{ chosenSource, chosenInstanceType, chosenNumOfInstances }] =
+    useWizardContext();
+  const dispatch = useDispatch();
+  const { mutate } = useMutation(createAWSReservation, {
+    onError: (error) => handleAlert(error),
+    onSuccess: () => handleAlert(),
+  });
+
+  const handleAlert = (error) => {
+    dispatch(
+      addNotification({
+        variant: error ? 'danger' : 'success',
+        title: error
+          ? 'an error has been occurred'
+          : 'Provisioning has been started',
+        description: error
+          ? `Provisioning cannot be started due to an error: ${error}`
+          : 'a notification will be trigger after a success or failure',
+      })
+    );
+  };
+
+  // TODO: add pubkey_id
+  const onComplete = () => {
+    mutate({
+      source_id: chosenSource,
```

Comment:
Can you add the parsing here? 
```js
    var id = parseInt(chosenSource)
    mutate({
      source_id: id,
```

Reply:
so we actually don't want to parse and keep it a string?